main/inc/lib/fileManage.lib.php

```diff
@@ -192,6 +192,9 @@ function move($source, $target, $forceMove = true, $moveContent = false)
 
         /* File case */
         if (is_file($source)) {
+            if (is_file($target . '/' . $file_name)) {
```

Comment:
It is the !$isWindowsOS condition that causes Windows system not to use the exec method because rm and mv are linux commands. 
ForceMove=true for Linux is fine and it now checks if a file or folder allready exists.